src/pages/PersonPage/PersonPage.tsx

```diff
@@ -20,12 +22,15 @@ const PersonPage: FC = () => {
   const { firebaseConfig } = useAppConfigContext();
 
   // Initialize the state of fetching the person's data
-  const { state: personDataState, dispatch: personDataDispatch } = useFetchData<Person>({
+  const { state: personDataState, dispatch: personDataDispatch } = useFetchData<PersonPageData>({
     isLoading: false,
```

Comment:
Similarly this will eventually change to

```
const fetchPersonData = useCallback(async () => {
    const personService = new PersonService(firebaseConfig);
    const voteService = new VoteService(firebaseConfig);
    const mattersSponsoredService = new MatterSponsorService(firebaseConfig);
    const rolesService = new RoleService(firebaseConfig);
    const [person, votes, mattersSponsored, roles] = await Promise.all([
      personService.getPersonById(id),
      voteService.getFullyPopulatedVotesByPersonId(id),
      mattersSponsoredService.getMattersSponsoredByPersonId(id),
      rolesService.getPopulatedRolesByPersonId(id),
    ])
    return Promise.resolve({
      person,
      votes,
      mattersSponsored,
      roles,
    })

  }, [id, firebaseConfig])

  // Initialize the state of fetching the person's data
  const { state: personDataState, dispatch: personDataDispatch } = useFetchData<PersonPageData>(
    {
      isLoading: false,
      error: null,
      hasFetchRequest: true,
    },
    fetchPersonData
  );
```

Reply:
I'm going to have to see this new paradigm in action before implementing it myself.